docs/src/modules/components/backers-zh.md

```diff
@@ -0,0 +1,41 @@
+## Supporting Material-UI
+
+Material-UI is an MIT-licensed open source project.
+It's an independent project with ongoing development made possible entirely
+thanks to the support of these awesome [backers](/discover-more/backers/).
+
+### Gold Sponsors
```

Comment:
💯 for translating the first `##` section. But I'm wondering about this one. Does it worth the maintenance overhead? To put into perspective with the relative low update frequency of the list. Also, it could be used to change the branding based on the language 🤔.